src/main/java/com/urswolfer/intellij/plugin/gerrit/push/GerritPushExtensionPanel.java

```diff
@@ -21,25 +21,32 @@
 import com.google.common.base.Strings;
 import com.google.common.collect.Lists;
 import com.google.common.collect.Maps;
+import com.intellij.openapi.project.ProjectManager;
 import com.intellij.uiDesigner.core.GridConstraints;
 import com.intellij.uiDesigner.core.GridLayoutManager;
 import com.intellij.util.ui.UIUtil;
+import org.apache.commons.lang.StringUtils;
```

Comment:
I prefer to use the Guava Joiner in this project (or when a project depends on a recent JDK of course native functionalities).

Reply:
Thanks for pointing me to Guava Joiner. It provides a bit prettier API for string joining purposes.